src/component/cards/cards.jsx

```diff
@@ -9,15 +10,15 @@ import './cards.pcss';
 const Cards = ({ cards }) => {
   return  <div className="cards">
     {cards.data.map(({ title, teaserIcon, teaserText, links }, i) => (
-      <a key={i} title={title} href={links.get('canonical').href}>
+      <Link key={i} title={title} href={links.get('canonical').href} children={
```

Comment:
children is a special prop- it's like a "slot" so instead of writing out the prop you can:



```
<Link normalProp={value}>
  <p>any markup</p>
 </Link> 
  ```
anything inside the element is the "children" 
https://react.dev/learn/passing-props-to-a-component#passing-jsx-as-children